src/App.js

```diff
@@ -1,24 +1,43 @@
-import logo from './logo.svg';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Toaster } from 'react-hot-toast';
+import { currentUser } from './redux/auth/currentUserSlice';
 import './App.css';
+import Login from './components/auth/Login';
+import SignUp from './components/auth/SignUp';
+import Home from './components/home/Home';
+import CarsIndex from './components/cars/CarIndex/CarIndex';
+import CarShow from './components/cars/CarShow/CarShow';
+import Layout from './components/layout/Layout';
+import AddNewCar from './components/cars/AddNewCar/AddNewCar';
+import AddReservation from './components/reservation/AddReservation';
+import ReservationList from './components/reservation/ReservationList';
+import CarDelete from './components/cars/CarDelete/CarDelete';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 8):

![image](https://github.com/muneeb3123/Fast-Lane-Racers-front-end/assets/57421823/d7858b78-6d8b-42e8-8e0b-a9e85142fbe4)


That way, your code will meet the latest standards. 💪